diff --git a/source/game_sa/General.h b/source/game_sa/General.h
index ff6f27f8..e528c06f 100644
--- a/source/game_sa/General.h
+++ b/source/game_sa/General.h
@@ -19,8 +19,8 @@ constexpr float RAND_MAX_FLOAT_RECIPROCAL = 1.0f / static_cast<float>(RAND_MAX);
 //#define BETTER_RNG
 
 namespace CGeneral { // More like `Math` (Or `Meth`, given how bad the code is, these guys must've been high!)
-    static thread_local std::random_device randomDevice{};
-    static thread_local std::mt19937 randomEngine(randomDevice());
+    static /*thread_local*/ std::random_device randomDevice{};
+    static /*thread_local*/ std::mt19937 randomEngine(randomDevice());
 
     void InjectHooks();
 
diff --git a/source/game_sa/MemoryMgr.cpp b/source/game_sa/MemoryMgr.cpp
index ebf88a45..4f7d6dea 100644
--- a/source/game_sa/MemoryMgr.cpp
+++ b/source/game_sa/MemoryMgr.cpp
@@ -438,18 +438,18 @@ void operator delete(void* memory) {
 
 #endif
 
-void* operator new(size_t size) {
-    return plugin::CallAndReturn<void*, 0x82119A, size_t>(size);
-}
+// void* operator new(size_t size) {
+//     return plugin::CallAndReturn<void*, 0x82119A, size_t>(size);
+// }
 
-void* operator new[](size_t size) {
-    return plugin::CallAndReturn<void*, 0x821195, size_t>(size);
-}
+// void* operator new[](size_t size) {
+//     return plugin::CallAndReturn<void*, 0x821195, size_t>(size);
+// }
 
-void operator delete(void* p) {
-    plugin::Call<0x8214BD, void*>(p);
-}
+// void operator delete(void* p) {
+//     plugin::Call<0x8214BD, void*>(p);
+// }
 
-void operator delete[](void* p) {
-    plugin::Call<0x8213AE, void*>(p);
-}
+// void operator delete[](void* p) {
+//     plugin::Call<0x8213AE, void*>(p);
+// }
